packages/disputer/README.md

```diff
@@ -0,0 +1,26 @@
+# @uma/disputer
+
+This package contains the UMA Dispute bot reference implementation. This executable will watch an ExpiringMultiParty
+contract for liquidations and dispute any that it deems to be invalid.
+
+## Installing the package
+
+```bash
+yarn global add @uma/disputer
+```
+
+## Running the disputer
+
+The simplest way to run the disputer (with default parameters and price feeds) is:
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=your.node.url MNEMONIC="your mnemonic here" disputer --network mainnet_mnemonic
```

Comment:
Will this not require `yarn` in front of the `disputer` command?

Reply:
also for setting these envs do you not need to use the `export` keyword?
should you chain them using `&&` or perhaps split this onto multiple lines for readability?

Reply:
1. It's related to your global question above. The disputer and liquidator are both packages that are primarily used to _run a script_ rather than import. You can install these packages in a directory, but my understanding is that most packages recommend global installation so the runnable file is available in the user's shell from anywhere. I don't have super strong opinions though. We could also encourage users to install in a folder and run from there -- the only downsides to that are that yarn will create a package.json, yarn.lock, and node_modules if the user isn't operating within an existing npm package.
2. You can pass env variables to a single command by just setting them as follows:
```bash
SOME_VAR="SOME_VALUE" some_command
```

I think this is slightly cleaner than export since it scopes the variables more tightly and their purpose is more explicit.

Reply:
1. I personally think that it's better to not install packages globally; especially for this kind of thing where you are required to have a `.env` file within the execution environment.
2. cool, that's pretty clean!

Reply:
1. Sounds good -- will update.
2. SG!

Reply:
Done.